baybe/utils/basic.py

```diff
@@ -128,3 +128,18 @@ def filter_attributes(
         for p in params
         if (p not in ignore) and hasattr(object, p)
     }
+
+
+class classproperty:
+    """A decorator to make class properties.
+
+    A class property combines the characteristics of @property and @classmethod. The
+    simple chaining of these two existing decorators is deprecated and causes mypy
+    issues.
```

Comment:
always good to have a link here to the deprecation (i.e. in the python changelog) and perhaps other pages that you considered helpful, for later reference

Reply:
added